js/src/useFlow.ts

```diff
@@ -13,12 +13,12 @@ function useFlow<T, T1, T2>(
   arg2: T2,
 ): T | null;
 function useFlow<T>(next: NextX<T>, ...args: any[]) {
-  const memoizedArgs = useMemoArray(args);
+  const memoizedArgs = useMemoArray(args.map((arg) => JSON.stringify(arg)));
```

Comment:
this does not work entirely because kotlin types like `Int` and `Double` are not expected as serialized in `next`

Reply:
resolved via `serializeArg`